### PR TITLE
Fix all compiler warnings.

### PR DESCRIPTION
**Pull request by Bren2010**
Previously we used [[ ... ]] for comments, and the body of the comment was only getting elided because the compiler didn't know what that syntax meant.

### CODE REVIEW

**Comment by Bren2010** [comment]
Please don't remove the .editorconfig file again! It's a [normal thing](https://github.com/quicwg/base-drafts/blob/master/.editorconfig) for projects to have!!

File: draft-ietf-mls-protocol.md [hunk 1]
```diff
@@ -2732,8 +2724,8 @@ New ciphersuite values are assigned by IANA as described in
 
 ## MLS Designated Expert Pool {#de}
 
-[[ OPEN ISSUE: pick DE mailing address.
-Maybe mls-des@ or mls-de-pool. ]]
+<!-- OPEN ISSUE: pick DE mailing address.
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md:2727]: I don't think this quite matches the intent -- we want these OPEN ISSUES to be visible in the rendered HTML

File: draft-ietf-mls-protocol.md [hunk 2]
```diff
@@ -2610,8 +2602,8 @@ tree is secret and thus so are all values derived from it.
 
 Initial leaf keys are known only by their owner and the group creator,
 because they are derived from an authenticated key exchange protocol.
-Subsequent leaf keys are known only by their owner. [[TODO: or by
-someone who replaced them.]]
+Subsequent leaf keys are known only by their owner.
+<!-- TODO: or by someone who replaced them. -->
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md:2606]: I think we can delete this TODO.

File: draft-ietf-mls-protocol.md (hunk 1 above)
Comment (Bren2010) [reply draft-ietf-mls-protocol.md:2727]: I noticed this is how QUIC WG does it: https://github.com/quicwg/base-drafts/commit/09338d0e237e67c648aa172c32b872e10bf737ff

**Pull request merged**